Header header

float64 longAccel
float64 longVel
float64 longPos
float64 latDist
float64 existProb
bool valid
